pages/start.tsx

```diff
@@ -242,12 +166,12 @@ const LoginStep3: React.FC<LoginStep3Props> = ({ interests, user }) => {
           "Content-Type": "application/json",
           Authorization: `Bearer ${sessionToken}`,
         },
-        body: JSON.stringify({ interests, timezone }),
+        body: JSON.stringify({ interests: [], timezone }),
```

Comment:
this is because we replace interests now with go to dev card but still need a value set for interests?

Reply:
Yes, its still required in the API for now

Reply:
I created https://github.com/open-sauced/api/issues/1030 for this.